src/main/java/at/hannibal2/skyhanni/data/MaxwellAPI.kt

```diff
@@ -17,6 +18,7 @@ import at.hannibal2.skyhanni.utils.StringUtils.removeResets
 import at.hannibal2.skyhanni.utils.StringUtils.trimWhiteSpace
 import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
 import net.minecraft.item.ItemStack
+import net.minecraftforge.fml.common.eventhandler.EventPriority
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 
 object MaxwellAPI {
```

Comment:
Two things
1. it doesnt update when Hypixel auto adjusts your Tunigs (§aYour §r§eTuning Points §r§awere auto-assigned as convenience!), it seems to spread out the tuning points equally between all selected slots but yeah idk just tell the player to reopen the inventory
2. It doesnt update when you change the tuning points inside of the "Stats Tuning" inventory, you should be able to use the top middle slot for updating the stats tho (look into dms for the image)

Reply:
already working on 2.

Reply:
for 1. i think showing a chat message to reopen for exact values is sufficient imo

Reply:
both done